sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -38,6 +39,10 @@
     <PackageReference Include="System.Net.Http" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
```

Comment:
In a comment, you said: 
> it would be good to add another .NET Framework TFM, i.e. net472 that drops that package dependency.

In general, we would like to avoid multi-targeting in Azure.Core, because of the complexity it adds, so it would be helpful for us to understand the need for this before merging this change.

Could you share more about why this is needed?

Reply:
Initially I thought that the net461 build can drop the System.Net.Http package dependency but I was wrong on that. The System.Net.Http package dependency is needed on .NET Framework < 4.7.1. Because of that, I introduced another .NET Framework TFM (net472 which is widely used in the stack) that drops the dependency.

This means that while you pay a little cost for building another inner build (net472), a consumer of Azure.Core doesn't transitively reference the System.Net.Http package anymore. Avoiding that package reference is important as System.Net.Http itself again brings in numerous other dependencies from the .NET Standard 1.x dependency graph.